package.json

```diff
@@ -55,6 +55,9 @@
     "vinyl": "2.X",
     "yargs": "7.X"
   },
+  "resolutions": {
+    "flush-write-stream": "1.0.3"
```

Comment:
you can just put this in devDeps and it'll be resolved to match the semver range of mississippi.

Reply:
reverted the accidental node 0.12 drop in flush-write-stream. fix out in 1.1.1

Reply:
Thank you